shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -10,7 +10,7 @@
   /* The time for which we'll disable launching other icons after tapping on
    * an app icon. This will also disable edit mode.
    */
-  const APP_LAUNCH_TIMEOUT = 3000;
+  const APP_LAUNCH_TIMEOUT = 1000;
```

Comment:
If you press home just after taping an icon on the homescreen (probably to tap another icon) the visibility of the homescreen stays the same. And without a visibility change the homescreen won't let you launch another app before `APP_LAUNCH_TIMEOUT`.
I'm open to suggestions if there's a better way to do this than lowering the timeout :)